frontend/src/presentation/pages/QuestionsPage.tsx

```diff
@@ -127,7 +126,10 @@ const QuestionsPage: React.FC = () => {
                         />
                         {selectedQuestionId && (
                             <div className={styles.addButtonWrapper}>
-                                <AddQuestionButton label={QUESTIONS_PAGE_TEXT.ADD_QUESTION} onClick={handleAddQuestion} />
+                                <AddQuestionButton
```

Comment:
Hmm, must the user select a question before they can access the add question button?

Reply:
No, if no question is selected, the add question button is in the landing panel. The Add Question button is always visible somewhere on the page.